network/ocsp.go

```diff
@@ -30,37 +30,37 @@ import (
 
 // Errors returned by CRL verifier
 var (
-	ErrInvalidConfigOCSPRequired   = errors.New("Invalid `ocsp_required` value")
-	ErrInvalidConfigOCSPFromCert   = errors.New("Invalid `ocsp_from_cert` value")
-	ErrInvalidConfigAllRequiresURL = errors.New("When passing `--tls_ocsp_required=all`, URL is mandatory")
-	ErrOCSPRequiredAllButGotError  = errors.New("Cannot query OCSP server, but --tls_ocsp_required=all was passed")
+	ErrInvalidConfigOCSPRequired   = errors.New("invalid `ocsp_required` value")
+	ErrInvalidConfigOCSPFromCert   = errors.New("invalid `ocsp_from_cert` value")
+	ErrInvalidConfigAllRequiresURL = errors.New("when passing `--tls_ocsp_required=all`, URL is mandatory")
+	ErrOCSPRequiredAllButGotError  = errors.New("cannot query OCSP server, but --tls_ocsp_required=all was passed")
 	ErrOCSPUnknownCertificate      = errors.New("OCSP server doesn't know about certificate")
-	ErrOCSPNoConfirms              = errors.New("None of OCSP servers confirmed the certificate")
+	ErrOCSPNoConfirms              = errors.New("none of OCSP servers confirmed the certificate")
 )
 
 // --tls_ocsp_required=<denyUnknown|allowUnknown|all>
 const (
 	// Deny certificates now known by OCSP server(s)
-	ocspRequiredYesStr = "denyUnknown"
+	ocspRequiredDenyUnknownStr = "denyUnknown"
 	// Allow certificates now known by OCSP server(s)
-	ocspRequiredNoStr = "allowUnknown"
+	ocspRequiredAllowUnknownStr = "allowUnknown"
 	// Effect of denyUnknown + all available OCSP servers (the one from config
 	// and those listed in certificate) should respond, otherwise deny the certificate
-	ocspRequiredAllStr = "all"
+	ocspRequiredRequireGoodStr = "requireGood"
 )
 
 var (
 	ocspRequiredValValues = map[string]int{
-		ocspRequiredYesStr: ocspRequiredYes,
-		ocspRequiredNoStr:  ocspRequiredNo,
-		ocspRequiredAllStr: ocspRequiredAll,
+		ocspRequiredDenyUnknownStr:  ocspRequiredDenyUnknown,
+		ocspRequiredAllowUnknownStr: ocspRequiredAllowUnknown,
+		ocspRequiredRequireGoodStr:  ocspRequiredReuireGood,
 	}
 )
 
 const (
-	ocspRequiredYes int = iota
-	ocspRequiredNo
-	ocspRequiredAll
+	ocspRequiredDenyUnknown int = iota
+	ocspRequiredAllowUnknown
+	ocspRequiredReuireGood
```

Comment:
Those names come from a department of redundancy department. (And have a "q" missing.)

Reply:
Renamed `ocspRequiredReuireGood` -> just `ocspRequiredGood`